internal/loadbalancebenchmark/communication.go

```diff
@@ -0,0 +1,34 @@
+package loadbalancingbenchmark
+
+import (
+	"fmt"
+)
+
+type EmptySignal struct{}
```

Comment:
We probably don't need this alias for such a basic type. It looks like we only use this type when we're dealing with channels. What _may_ be a better alias would be for a `chan struct{}` type

Reply:
removed that alias.